docs/readthedocs/source/doc/PythonAPI/Nano/pytorch.rst

```diff
@@ -18,10 +18,10 @@ bigdl.nano.pytorch.InferenceOptimizer
     :exclude-members: ALL_INFERENCE_ACCELERATION_METHOD, DEFAULT_INFERENCE_ACCELERATION_METHOD
     :inherited-members:
 
-bigdl.nano.pytorch.TorchNano
+TorchNano API
```

Comment:
Is it a reasonable title to include both `TorchNano` and `@nano` decorator?

Reply:
I think is's ok